proteuslib/unit_models/tests/test_reverse_osmosis_1D.py

```diff
@@ -461,27 +386,27 @@ def test_conservation(self, RO_frame):
     @pytest.mark.component
     def test_solution(self, RO_frame):
         m = RO_frame
-        x_interface_in = value(1 / m.fs.unit.nfe)
-        assert (pytest.approx(-1.5596e5, rel=1e-3) == value(m.fs.unit.deltaP_stage[0]))
-        assert (pytest.approx(1.219e-2, rel=1e-3) ==
+        x_interface_in = m.fs.unit.feed_side.length_domain[2]
+        assert (pytest.approx(-1.755e5, rel=1e-3) == value(m.fs.unit.deltaP_stage[0]))
+        assert (pytest.approx(7.992e-3, rel=1e-3) ==
```

Comment:
Why did the inlet flux change? Shouldn't that basically only be based on the inlet conditions and independent on the number of nodes?

Reply:
The flux at the "inlet" is not directly at the inlet. It is the second element in the continuous set (length_domain). So we should expect this value to change as it is not directly fixed by the inlet conditions. As a reminder, the actual inlet is skipped for most constraints, including those for membrane flux.